src/api/database/models/Group.ts

```diff
@@ -31,6 +31,14 @@ class Group extends ParanoidModel<
   @Column(STRING)
   meetingLink: string | null;
 
+  @AllowNull(true)
+  @Column(STRING)
+  meetingId: string | null;
+
+  @AllowNull(true)
+  @Column(DATE)
+  meetingLinkCreatedAt: Date | null;
```

Comment:
`meetingCreatedAt`

Reply:
Actually, consider using a single JSONB column `meeting` to encode all tencent meeting related information:

```
{ link: string, id: string, createdAt: Date }
```

because we don't need indexes into individual fields of this data structure, and it gives a nice abstraction / encapsulation layer of the data model.